packages/contracts/tests/js/StabilityPool_ZEROIssuanceTest.js

```diff
@@ -198,32 +196,32 @@ contract('StabilityPool - ZERO Rewards', async accounts => {
 
       // Get G and communityIssuance before
       const G_Before = await stabilityPool.epochToScaleToG(0, 0);
-      const ZEROIssuedBefore = await communityIssuanceTester.totalZEROIssued();
+      const ZEROIssuedBefore = await communityIssuanceTester.totalSOVIssued();
 
       //  A withdraws some deposit. Triggers issuance.
       const tx = await stabilityPool.withdrawFromSP(1000, { from: A, gasPrice: 0 });
       assert.isTrue(tx.receipt.status);
 
       // Check G and ZEROIssued do not increase, since <1 minute has passed between issuance triggers
       const G_After = await stabilityPool.epochToScaleToG(0, 0);
-      const ZEROIssuedAfter = await communityIssuanceTester.totalZEROIssued();
+      const ZEROIssuedAfter = await communityIssuanceTester.totalSOVIssued();
```

Comment:
here afterwards, the test verifies that no SOV have been issued because the time passed since the last issuance was < 60s.
We do not have any 60s checks anymore on the new issuance contract. so, how come that the assertion below still holds?

Reply:
In this test cases, there was no SOV issued, so it expect the the reward term G is also not updated.
The comment here seems confusing, i updated.